### ISSUE TITLE
Consider disabling IIS deployment scenario tests

### ISSUE THREAD

**Issue by DavidObando** [aspnet/Performance#114]
IIS scenario tests are failing due to IIS Deployer not being actively maintained. Here's the error we're getting:

```
    Startup [Variation: StarterMvc_IISScenario] [Stage: Collection] [Iteration: 1/1] [FAIL]
      System.NotSupportedException : Found no deployers suitable for server type 'IIS' with the current runtime.
      Stack Trace:
           at Microsoft.AspNetCore.Server.Testing.ApplicationDeployerFactory.Create(DeploymentParameters deploymentParameters, ILogger logger)
        D:\GitHub\aspnet\Performance\src\Benchmarks.Utility\Helpers\IISTestManager.cs(51,0): at Benchmarks.Utility.Helpers.IISTestManager.Initialize(ILoggerFactory loggerFactory)
        D:\GitHub\aspnet\Performance\test\Microsoft.AspNetCore.Tests.Performance\WebIISPerformance.cs(30,0): at Microsoft.AspNetCore.Tests.Performance.WebIISPerformance..ctor(IISTestManager testManager)
```

It's not being used in any of the product's automated tests since it requires running the process with admin rights, see aspnet/Hosting#684.

Consider disabling these tests **for now**. The team might fix the IIS Deployer, which would allow us to keep running these tests in the future.

**Comment by DavidObando** [comment]
Issue has been fixed, will verify soon. Currently blocked by issues related to move to shared framework.

**Comment by DavidObando** [comment]
Addressed by #133
